### PR TITLE
Move datasource to JSON

### PR DESCRIPTION
**Pull request by icco**
I generate this json file from https://www.uspresidentialelectionnews.com/2020-presidential-primary-schedule-calendar/

Also did some general cleanup throughout the files.

### CODE REVIEW

File: election_reminder.py [hunk 1]
```diff
@@ -1,34 +1,45 @@
...
-	d = contest['date'] - datetime.date.today()
+	dflt_tz = dateutil.tz.tzoffset("EST", -18000)
+	today = dateutil.utils.today(dflt_tz)
+	d = dateutil.parser.parse(contest['date']) - today
 	if d.days < 0:
-		continue #go to next contest
+			continue # go to next contest
 	try:
-		fact = contest['facts'][d.days]
+			fact = contest['facts'][d.days]
```
Comment (michaelefisher) [inline election_reminder.py]: I think something happened to the indentation here.

File: election_reminder.py [hunk 2]
```diff
@@ -1,34 +1,45 @@
+#! /usr/bin/env python
 # -*- coding: utf-8 -*-
+# vim: set noexpandtab ts=2:
```
Comment (michaelefisher) [inline election_reminder.py]: 🤔

**Comment by michaelefisher** [comment]
@icco can you rebase? I merged in travis stuff.

File: election_reminder.py (hunk 2 above)
Comment (icco) [reply election_reminder.py]: My vim config and your choice of whitespace in python were not getting along.

File: election_reminder.py (hunk 1 above)
Comment (icco) [reply election_reminder.py]: Fixed

**Comment by icco** [comment]
Fixed @michaelefisher !

**Review by michaelefisher — APPROVED** [review]
Tested locally, seems good to me:
```
  docker run -e SLACK_WEBHOOK_TOKEN=${SLACK_WEBHOOK_TOKEN} natops
Sending Payload:
{"icon_emoji": ":ghost:", "text": "It is Tuesday. Iowa Caucuses are in 377 days. ", "username": "ElectionReminder"}
```

**Pull request merged**